src/components/Card.tsx

```diff
@@ -0,0 +1,18 @@
+import '../scss/card.scss';
+
+type CardProps = {
+  className?: string;
+  children?: JSX.Element[] | JSX.Element;
```

Comment:
Would be better to use `React.ReactNode`, then you wouldn't need the non-null assertion in the group invite page https://github.com/Imageus-OSS/client/blob/f959647ce21abd79b771f54dbe5d6209cea87af6/src/pages/GroupInvitePage.tsx#L238